extensions/lifecycle/LifecycleConfigValidator.js

```diff
@@ -8,6 +8,10 @@ const joiSchema = {
     objectTasksTopic: joi.string().required(),
     auth: authJoi.required(),
     conductor: {
+        bucketSource: joi.string().
+            valid('bucketd', 'zookeeper').default('zookeeper'),
+        bucketd: hostPortJoi.
```

Comment:
noticed in https://github.com/scality/Federation/pull/3744/files#diff-eebf0415822c288651cc7eb695d029d6bfd390ac9bb4699be39a8cb76f59af67R126-R131 there is a reference to an `https` field.

`hostPortJoi` does not define that, so there may be a validation issue